### DISCUSSION TITLE
Grafana implementation in IFrame

### DISCUSSION THREAD

**Discussion by TomasAdvance** [frangoteam/FUXA#269] [General]
Dear developer,
Did you thinking about implementation of Grafana as iframe service (or separate object) where Grafana will used data from FUXA DB?
I sugest it may be bring very power full capabilities for FUXA in charts point of view and no own developing charts is needed.
I will be very grateful for your response, or I can help with ideas on how to implement this.
BR
Tomas

**Comment by unocelli** [comment]
Hi,
add a iframe in a card of multi view page shouldn't be very complicated, to add into the SVG as control needs to be verified #250.
The FUXA DB is sqlite, not really the best choice to save history tags values for several reasons. It would be better to save in a database like influxDB but this is an other todo task ;-)

**Reply by TomasAdvance** [reply]
I saw your reply #250. I understand what you mean. But if Grafana and for example influx DB will be used only for history data, I think will be very power full tool for FUXA. I know you are use SQlitle, for that Grafana have own plug-in (https://grafana.com/grafana/plugins/frser-sqlite-datasource/) what are able implemented SQlitle.
What do you think about that?

**Reply by unocelli** [reply]
of course it could work, I just have some reservations about the performance aspect when there is a lot of data.
another solution I have already heard is to save fuxa data to influxdb via mqtt and then access from grafana to influxdb

**Reply by TomasAdvance** [reply]
It sounds like interested solution
